src/components/artwork/details.tsx

```diff
@@ -19,7 +19,7 @@ export class ArtworkDetails extends React.Component<DetailsProps, null> {
     } else if (artists && artists.length) {
       let artistsEl = []
       for (let i = 0; i < artists.length; i++) {
-        artistsEl.push(<TextLink href={ artists[i].href }>{ artists[i].name }</TextLink>)
+        artistsEl.push(<TextLink href={ artists[i].href } key={ i }>{ artists[i].name }</TextLink>)
```

Comment:
For more info see [the docs](https://facebook.github.io/react/docs/lists-and-keys.html#keys) and e.g. a blog post like [this](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318#.wvko033lh)